### PR TITLE
update  iotsharp.data.taos

### PR DESCRIPTION
**Pull request by maikebing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [DG-IoT](https://github.com/dgiot/dgiot-dashboard) - DG-IoT platform industry application extension plugin.
 - [dgiot_tdengine](https://github.com/dgiot/dgiot_tdengine) - TDengine adapter plugin.
 - [node-red-contrib-tdengine](https://github.com/kp45-tech/node-red-contrib-tdengine) - TDengine connector plugin for Node-RED.
+- [IoTSharp](https://github.com/IoTSharp/IoTSharp/blob/master/IoTSharp.Data.TimeSeries/TaosStorage.cs) - IoTSharp is an open-source IoT platform for data collection,.It supports storing time series data to TDengine.
```
Comment (ericyangpan) [inline README.md]: There is a typo after collection

File: README.md [hunk 2]
```diff
@@ -55,6 +56,7 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [crown](https://github.com/machine-w/crown) - crown is a simple and small ORM for TDengine.
 - [APIJSON](https://github.com/Tencent/APIJSON) - A JSON Transmission Protocol and an ORM Library for providing APIs and Docs without writing any code.
 - [ZORM](https://gitee.com/chunanyong/zorm) - ORM support TDenging by Go.
+-
```
Comment (ericyangpan) [inline README.md]: Remove this empty item.

File: README.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [DG-IoT](https://github.com/dgiot/dgiot-dashboard) - DG-IoT platform industry application extension plugin.
 - [dgiot_tdengine](https://github.com/dgiot/dgiot_tdengine) - TDengine adapter plugin.
 - [node-red-contrib-tdengine](https://github.com/kp45-tech/node-red-contrib-tdengine) - TDengine connector plugin for Node-RED.
+- [IoTSharp](https://github.com/IoTSharp/IoTSharp/blob/master/IoTSharp.Data.TimeSeries/TaosStorage.cs) - IoTSharp is an open-source IoT platform for data collection,It supports storing time series data to TDengine.
```
Comment (ericyangpan) [inline README.md]: A space followed comma is needed.

File: README.md [hunk 4]
```diff
@@ -70,6 +72,12 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [TDengine Rust bindings](https://github.com/songtianyi/tdengine-rust-bindings) - It's a Rust bindings project for TDengine.
 - [TDenginex](https://github.com/lizhaochao/TDenginex) - TDengine Connector for Elixir.
 - [DBUtility.TDengine](https://github.com/cockroach888/GSA.MOLLE.ToolKits/tree/main/src/ToolKits.DBUtility) - C# Connector for TDengine. (.NET 6+)
+- [IoTSharp.Data.Taos](https://www.nuget.org/packages/IoTSharp.Data.Taos/)  - TDengine's ADO.Net provider. (.Net Framework 4.6+, .Net Standard2.0+,.Net 5.0+)  [more](https://github.com/IoTSharp/EntityFrameworkCore.Taos)
```
Comment (ericyangpan) [inline README.md]: [more] link cannot appear in the description

Comment (ericyangpan) [reply README.md]: .Net or .NET?

File: README.md [hunk 5]
```diff
@@ -70,6 +72,12 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [TDengine Rust bindings](https://github.com/songtianyi/tdengine-rust-bindings) - It's a Rust bindings project for TDengine.
 - [TDenginex](https://github.com/lizhaochao/TDenginex) - TDengine Connector for Elixir.
 - [DBUtility.TDengine](https://github.com/cockroach888/GSA.MOLLE.ToolKits/tree/main/src/ToolKits.DBUtility) - C# Connector for TDengine. (.NET 6+)
+- [IoTSharp.Data.Taos](https://www.nuget.org/packages/IoTSharp.Data.Taos/)  - TDengine's ADO.Net provider. (.Net Framework 4.6+, .Net Standard2.0+,.Net 5.0+)  [more](https://github.com/IoTSharp/EntityFrameworkCore.Taos) 
+
+
+## Health Checks
```
Comment (ericyangpan) [inline README.md]: I don't think `Health Checks` can be an independent category now. You can place it in `Others` section till there are more content under `Health Checks`.

**Review by ericyangpan — CHANGES_REQUESTED** [review]
See the single comments.

**Comment by maikebing** [comment]
合并？

**Pull request merged**